week3.ipynb

```diff
@@ -0,0 +1,495 @@
+{
```

Comment:
<p>I would suggest something like "Follow the instructions below, Steps 1, 2, 3, etc. will help you do <strong><em>_</em></strong>________________________.</p><p> </p><p>It seems a bit like hand-holding but coding can be intimidating! </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/CUB-Computational-Tools/2020-ps-vaga3461/pull/4/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>